tests/pipelines/animatediff/test_animatediff_video2video.py

```diff
@@ -127,6 +133,36 @@ def get_dummy_inputs(self, device, seed=0):
         }
         return inputs
 
+    def test_from_pipe_consistent_config(self):
+        assert self.original_pipeline_class == StableDiffusionPipeline
```

Comment:
We only made the model size smaller here, for AnimateDiff, no? It can definitely be applied across everything though to gain a good speed up. In fact, we can still make the model slower with 1 layer per block instead of 2, smaller block out channels, smaller cross attn, etc. Will be some effort if we'd like to do it, but always good to have these tests running faster IMO

Reply:
I think for now it will be pretty easy and safe to update the from pipe tester mixin here https://github.com/huggingface/diffusers/blob/1fd647f2a0e97b3746a0a038a0b826f3c408422a/tests/pipelines/test_pipelines_common.py#L654

Reply:
Won't updating that cause most, if not all, tests inheriting from PipelineFromPipeTesterMixin to fail?

For example, in the test `test_from_pipe_consistent_config` we use `hf-internal-testing/tiny-stable-diffusion-pipe` which has larger model sizes than the one we're using here (`hf-internal-testing/tinier-stable-diffusion-pipe`).

In order to use the latter, we will have to update all model configurations in all tests as well as expected_slice values.